runtime/src/lib.rs

```diff
@@ -93,19 +93,19 @@ pub type UncheckedExtrinsic = generic::UncheckedExtrinsic<Address, Call, Signatu
 
 parameter_types! {
   pub const BlockHashCount: BlockNumber = BLOCK_HASH_COUNT;
-  pub const BondDuration: u32 = 1;
+  pub const BondDuration: u32 = 7 * BLOCKS_PER_DAY as u32;
```

Comment:
```suggestion
  pub const BondDuration: u32 = BLOCKS_PER_DAY as u32;
```

Shorten it for the testnet so we can change the collator set more often.